src/api/app/models/package.rb

```diff
@@ -812,22 +812,6 @@ def activity
     package.shift.activity_value.to_f
   end
 
-  # is called before_update
-  def update_activity
-    # the value we add to the activity, when the object gets updated
-    addon = 10 * (Time.now.to_f - updated_at_was.to_f) / 86400
-    addon = 10 if addon > 10
-    new_activity = activity + addon
-    new_activity = 100 if new_activity > 100
-
-    # rails 3 only - rails 4 is reported to name it update_columns
-    update_column(:activity_index, new_activity)
-    # we need to update the timestamp manually to avoid the activity_algorithm to run away
-    update_column(:updated_at, Time.now)
```

Comment:
well, I didn't invented this statistics controller - I only fixed it. Removing 'the test' implies removing the statistics controller - without the tests it's just outputting numbers.